presto-main/src/main/java/com/facebook/presto/server/TaskUpdateRequest.java

```diff
@@ -63,6 +66,24 @@ public TaskUpdateRequest(
         this.tableWriteInfo = tableWriteInfo;
     }
 
+    public TaskUpdateRequest(
```

Comment:
This constructor is only used in `HttpRemoteTask`  right? Should we just inline it?